db/upgrade_migrations/20170125033319_upgrade_v0_9_to_v0_10.rb

```diff
@@ -1,9 +1,28 @@
 # frozen_string_literal: true
 class UpgradeV09ToV010 < ActiveRecord::Migration
-  def change
+  def up
     remove_foreign_key :thredded_messageboard_users, :thredded_messageboards
     add_foreign_key :thredded_messageboard_users, :thredded_messageboards, on_delete: :cascade
     remove_foreign_key :thredded_messageboard_users, :thredded_user_details
     add_foreign_key :thredded_messageboard_users, :thredded_user_details, on_delete: :cascade
+
+    create_table :thredded_user_post_notifications do |t|
+      t.references :user, null: false
```

Comment:
FeatureEnvy: UpgradeV09ToV010#up refers to 't' more than self (maybe move it to another class?)